src/styles/main.scss

```diff
@@ -0,0 +1,97 @@
+$stopwatchSize: 500px;
+$minuteHandSize: $stopwatchSize/2;
+$minuteHandWidth: 5px;
+$secondHandSize: $stopwatchSize/3;
+$secondHandWidth: 5px;
+$nailSize: $minuteHandWidth * 3;
+
+* {
+  box-sizing: border-box;
+}
+
+.stopwatch {
+  width: $stopwatchSize;
+  height: $stopwatchSize;
+  border-radius: 50%;
+  border: 1px dotted #000;
+  margin-inline: auto;
+  margin-block: calc((100vh - #{$stopwatchSize}) / 2);
+
+  &__minute-hand {
+    width: $minuteHandWidth;
+    height: $minuteHandSize;
+    background-color: #888;
+    position: relative;
+    margin: 0 auto;
+    z-index: 1;
+    transform-origin: center bottom;
+    animation: moveMinutes 5s ease-in-out infinite;
+
+    &::after {
+      content: "";
+      position: absolute;
+      width: $nailSize;
+      height: $nailSize;
+      border-radius: 50%;
+      border: 1px solid #555;
+      background-color: #555;
+      bottom: calc((0% - #{$nailSize}) / 2);
+      left: calc((50% - #{$nailSize}) / 2);
+    }
```

Comment:
It's a center circle. It's not a pseudo-element for minute-hand . It looks to be a pseudo-element for stopwatch

Reply:
done